app/src/main/res/layout-land/topic_practice_header_view.xml

```diff
@@ -19,24 +19,23 @@
 
     <TextView
       android:id="@+id/master_skills_text_view"
+      style="@style/Heading2"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
```

Comment:
layout_width and layout_height can be removed from this and other files as it is `wrap_content`

Reply:
Done.